util/prep-bazel-airgapped-build.sh

```diff
@@ -143,10 +143,10 @@ if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
     //... \
     @remote_java_tools//... \
     @remote_java_tools_linux//... \
-    @bindgen_clang_linux//... \
+    @bindgen_clang_linux_$(uname -m)//... \
```

Comment:
Similarly, should we use a variable here instead of expanding `uname` individually.